### DISCUSSION TITLE
any way to move a directory tree to the other drive?

### DISCUSSION THREAD

**Discussion by TomW1605** [trapexit/mergerfs#1015] [Q&A] [answered]
i recently set up some new drives with mergerfs and snapraid to replace a full raid5 array. i just finished copying my plex library onto the new drives. the drives are ~10TiB and the plex library is ~11TiB so it completely filled the first drive. because of this i now have 1 full drive with 90% of the media and 1 mostly empty drive with only some of it (the only media here is a sub set of my tv shows). this also means i needed to change the policy from `epmfs ` to `mfs` so that sonarr could still add episodes to the tv folder.

is there a way i can move the tv folder completely to the second drive? that way i can have one that is dedicated to TV and the other that has everything else (the tv is about as big as everything else)

**Comment by trapexit** [comment]
I'm not sure I really follow. Using a non-path preserving policy is in contradiction with having separate drives for separate things.

**Reply by TomW1605** [reply]
i want to use a path preserving policy but i cant because the drive is full (see my other question). so i want to move the TV shows onto the other drive so i can switch back to a path preserving policy

**Comment by trapexit** [comment, marked as answer]
OK. Then move the files to the other filesystem.

**Reply by TomW1605** [reply]
can i just move it with `mv` on the physical disk? that wont mess up the merged disk?

**Reply by trapexit** [reply]
Yes. People do this all the time. It's shown in the docs. It's done by the mergerfs-tools. Everyone merging cloud storage does it. Besides possible caching issues (which depends entirely on your setup and usage pattern) mergerfs couldn't care less. As the docs say it's just a proxy. You can add and remove stuff at any time.

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs#can-drives-be-written-to-directly-outside-of-mergerfs-while-pooled

**Comment by azohra** [comment]
I wrote a tool for this exact use case today and I hope it helps you out.

https://github.com/azohra/unbalance
